leapp/libraries/stdlib/__init__.py

```diff
@@ -32,3 +67,51 @@ def call(args, split=True):
     if split:
         return r.splitlines()
     return r
+
+
+def _logging_handler(fd_info, buffer):
+    (_unused, fd_type) = fd_info
+    if os.getenv('LEAPP_DEBUG', '0') == '1':
+        if fd_type == call.STDOUT:
+            sys.stdout.write(buffer)
+        else:
+            sys.stderr.write(buffer)
+
+
+def new_call(args):
+    """
+    Call the Leapp's stdlib _call
```

Comment:
That’s not a good docstring, it tells me only something about the usage of an internal function.

Better would be perform something like:
Run the program described by args and wait for the program to complete and return the results as a dict. The execution of the program and it’s results are captured by the audit.